src/encoded/static/scss/encoded/modules/_immune_cells.scss

```diff
@@ -0,0 +1,238 @@
+$bubbleSize: 13px;
```

Comment:
This is another philosophical discussion. Should we use css var or stick to SCSS/SASS variables-style. I prefer SCSS/SASS; but one could argue to always take inbuilt over third party.